pf9/cluster/commands.py

```diff
@@ -36,9 +36,12 @@ def run_express(ctx, inv_file, ips):
     log_file = os.path.join("/var/log/pf9",
                             datetime.now().strftime('express_%Y_%m_%d-%H_%m_%S.log'))
     # Invoke PMK only related playbook.
-    cmd = 'sudo {0} -a -b --pmk -v {1} -c {2} -l {3} pmk'.format(exp_ansible_runner,
-                                                inv_file, exp_config_file, log_file)
-
+    # Should this be defined directly in the inv file template? It could be a global setting
+    # for CLI inventory files.
+    ansible_extra_vars = "\"custom_py_interpreter=/opt/pf9/cli/bin/python\""
+    cmd = 'sudo {0} -a -b --pmk -v {1} -c {2} -l {3} -e {4} pmk'.format(exp_ansible_runner,
```

Comment:
are there tests running this functionality?
may be a good to add for later.

Reply:
Not at the moment. But we need to add tests.